IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CableSegmentTypeContactWire/DocPropertySet.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CableSegmentTypeContactWire" UniqueId="62b5e37b-a136-0179-2ff7-18057ba3177f" ApplicableType="IfcCableSegment/CONTACTWIRESEGMENT,IfcDistributionSystem/OVERHEAD_CONTACTLINE_SYSTEM" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
Is system an applicable Type?

Reply:
No! Agree with you! Commit : 305806220fb93c9010687f137214b3f48df310cb